### ISSUE TITLE
NullReferenceException on trying to create a ticket

### ISSUE THREAD

**Issue by akiraveliara** [InsanityBot/insanitybot-v2#97]
InsanityBot fails to replace all values for the new ticket embed - the ticket is created regardless. It can be presumed this is an async issue/race condition, since it only happens in slower environments and even there is very inconsistent.

To reproduce, create a ticket - see above

This happens both on Pterodactyl and Windows, unable to reproduce on Linux.
Bot version: 2.0.0-dev.00035 and onwards.

**Comment by akiraveliara** [comment]
It should also be noted only the [direct replace parser](https://github.com/InsanityBot/insanitybot/blob/main/InsanityBot/Tickets/Placeholders/Parser.cs#L21-L28) is affected. The [character-by-character parser](https://github.com/InsanityBot/insanitybot/blob/main/InsanityBot/Tickets/Placeholders/Parser.cs#L32-L88) handling more complex replacement is unaffected.

**Comment by akiraveliara** [comment]
Still happens rarely - in fact i was unable to reproduce it at all in any other environment

**Comment by akiraveliara** [comment]
~~~cs
[2021-08-05 12:34:17 +02:00] [Error] [1001/CommandError] Command: new failed:
System.NullReferenceException: Object reference not set to an instance of an object.
   at InsanityBot.Tickets.Commands.LangPlaceholders.ReplaceValues(String value, CommandContext context, DiscordChannel ticket) in C:\projects\insanitybot\InsanityBot\Tickets\Commands\LangPlaceholders.cs:line 16
   at InsanityBot.Tickets.Commands.NewTicketCommand.NewTicket(CommandContext ctx, String data) in C:\projects\insanitybot\InsanityBot\Tickets\Commands\NewTicketCommand.cs:line 47
   at DSharpPlus.CommandsNext.Command.ExecuteAsync(CommandContext ctx): Object reference not set to an instance of an object.
   at InsanityBot.Tickets.Commands.LangPlaceholders.ReplaceValues(String value, CommandContext context, DiscordChannel ticket) in C:\projects\insanitybot\InsanityBot\Tickets\Commands\LangPlaceholders.cs:line 16
   at InsanityBot.Tickets.Commands.NewTicketCommand.NewTicket(CommandContext ctx, String data) in C:\projects\insanitybot\InsanityBot\Tickets\Commands\NewTicketCommand.cs:line 47
   at DSharpPlus.CommandsNext.Command.ExecuteAsync(CommandContext ctx)
[2021-08-05 12:34:51 +02:00] [Info] [1100/StopCommand] Shutting down InsanityBot v2.0.0-dev.00036...
~~~

**Comment by akiraveliara** [comment]
This issue has now been fixed for windows - for all we know, at any rate

**Comment by akiraveliara** [comment]
fixed as of 457e1e7
